cmd/aries-js-worker/index.html

```diff
@@ -31,7 +31,7 @@
             const operation = document.querySelector("#operation").value
             const method = document.querySelector("#method").value
             const response = await aries[operation][method](request)
-            document.querySelector("#output").value = response
+            document.querySelector("#output").value = JSON.stringify(response)
```

Comment:
I wonder if it's possible to hide stringify in the library. https://github.com/hyperledger/aries-framework-go/blob/master/cmd/aries-js-worker/src/aries.js#L60